client/app/components/LoadMoreButton/LoadMoreButton.scss

```diff
@@ -1 +1,6 @@
-@import "~styles/_global.scss"
\ No newline at end of file
+@import "~styles/_global.scss";
+
+.container {
+	text-align: center;
```

Comment:
Good catch with the `center` tag!

Just a minor nitpick here: this indentation seems to be using a tab, rather than the 2 spaces mentioned in the [frontend practices page](https://github.com/ifmeorg/ifme/wiki/Frontend-Practices#code-quality).

In any case, I think there might be a follow-up PR for stylelint fixes that could address this too?

Reply:
Ah yeah, my IDE has tabs as default for work. I'll fix that.